mutable_tree.go

```diff
@@ -951,12 +965,44 @@ func (tree *MutableTree) saveFastNodeVersion() error {
 	return tree.ndb.setFastStorageVersionToBatch()
 }
 
+func (tree *MutableTree) handleOrphans(version int64) error {
```

Comment:
is this function threadsafe?

Reply:
it's not threadsafe, but it would not run concurrently